<launch>
	<arg name="marker_size" default="8.0" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	
	<arg name="cam_image_topic" default="/kinect_head/depth_registered/points" />
	<arg name="cam_info_topic" default="/kinect_head/rgb/camera_info" />		
	<arg name="output_frame" default="/map" />
	

	<!--<arg name="cam_image_topic" default="/narrow_stereo/left/image_color" />
	<arg name="cam_info_topic" default="/narrow_stereo/left/camera_info" />		
	<arg name="output_frame" default="/map" /> 
	-->
	<!--<arg name="output_frame" default="/narrow_stereo_l_stereo_camera_optical_frame" />-->

	
	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
	

	<!-- Trying to use no kinect option 
	<arg name="marker_size" default="4.6" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/narrow_stereo/right/image_raw" />
	<arg name="cam_info_topic" default="/narrow_stereo/right/camera_info" />		
	<arg name="output_frame" default="/odom_combined" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
	-->

</launch>
